src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationByProfileIdsRequestValidatorImpl.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.professionalapi.controller.request.validator.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.professionalapi.controller.request.InvalidRequest;
+
+@Component
+public class OrganisationByProfileIdsRequestValidatorImpl {
+    public void validate(Integer pageSize) {
+        if (pageSize != null && pageSize < 1) {
+            throw new InvalidRequest("Invalid pageSize");
```

Comment:
This does not match the actual message described in the AC10
Similarly no specific test for AC11 message